README.md

```diff
@@ -38,7 +38,7 @@ jobs:
           GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
 
       - name: Create Issue From File
-        if: steps.lychee.outputs.exit_code != 0
+        if: failure()
```

Comment:
Either defer this PR change to 2.0, or also update `lychee` step to set `with: fail: true`.

```suggestion
        if: failure() && steps.lychee.outcome == 'failure'
```

**NOTE:** This example currently should only work properly if `steps.lychee.with.fail: true` (_the prior `exit_code` output did not require that_), until 2.0 release makes that default, or more correctly drops `fail` input and defers to `steps.lychee.continue-on-error: true` for the 2.0 `fail: false` behaviour.